### PR TITLE
Happy 7th birthday kubeadm!

### PR DESCRIPTION
**Pull request by fabriziopandini**
Thanks to @sftim who proposed the idea, this post is celebrating kubeadm's 7th birthday

/cc @sftim @pacoxu @neolit123

### CODE REVIEW

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:3]:
```suggestion
title: 'Happy 7th Birthday kubeadm!'
```
we use title case for article titles.

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:11]:
```suggestion
Starting from the initial blog post [“How we made Kubernetes insanely easy to install”](/blog/2016/09/how-we-made-kubernetes-easy-to-install/) in September 2016, followed by an exciting growth that lead to general availability / [“Production-Ready Kubernetes Cluster Creation with kubeadm”](https://kubernetes.io/blog/2018/12/04/production-ready-kubernetes-cluster-creation-with-kubeadm/) two years later.
```

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:4]: Is this the actual 7th birthday? I thought it would be from the v1.4 release day which was on a different date.

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:17]:
```suggestion
kubeadm is focused on bootstrapping Kubernetes clusters on existing infrastructure and performing an essential set of maintenance tasks. The core of the kubeadm interface is quite simple: new control plane nodes
are created by running [`kubeadm init`](/docs/reference/setup-tools/kubeadm/kubeadm-init/) and
worker nodes are joined to the control plane by running
[`kubeadm join`](/docs/reference/setup-tools/kubeadm/kubeadm-join/).
Also included are utilities for managing already bootstrapped clusters, such as control plane upgrades
and token and certificate renewal.
```

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:25]:
```suggestion
Infrastructure provisioning, for example, is left to other SIG Cluster Lifecycle projects, such as the
[Cluster API](https://cluster-api.sigs.k8s.io/). Instead, kubeadm covers only the common denominator
in every Kubernetes cluster: the
[control plane](/docs/concepts/overview/components/#control-plane-components).
The user may install their preferred networking solution and other add-ons on top of Kubernetes
*after* cluster creation.
```

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:55]:
```suggestion
- Try [kubeadm](/docs/setup/) to install Kubernetes today
```

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:27]:
```suggestion
Behind the scenes, kubeadm does a lot. The tool makes sure you have all the key components:
etcd, the API server, the scheduler, the controller manager. You can join more control plane nodes
for improving resiliency or join worker nodes for running your workloads. You get cluster DNS
and kube-proxy set up for you. TLS between components is enabled and used for encryption in transit.
```

**Review by sftim — COMMENTED** [review]
Thanks. Here are 7 items of feedback.

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (pacoxu) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:11]:
```suggestion
Starting from the initial blog post [“How we made Kubernetes insanely easy to install”](https://kubernetes.io/blog/2016/09/how-we-made-kubernetes-easy-to-install/) in September 2016, followed by an exciting growth that leads to general availability / [“Production-Ready Kubernetes Cluster Creation with kubeadm”](https://kubernetes.io/blog/2018/12/04/production-ready-kubernetes-cluster-creation-with-kubeadm/) two years later.
```

**Review by pacoxu — APPROVED** [review]
LGTM

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (ydFu) [inline content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:31]:
```suggestion
In all and for all kubeadm's story is tightly coupled with Kubernetes' story, and with this amazing community.
```

**Comment by neolit123** [comment]
looks like this has some basic nits.

@fabriziopandini this is well written, thank you.
the original Google Doc seemed fine to me already.

/cc @pacoxu 
leaving kubeadm maintainer `/lgtm` to you. thanks!

**Comment by sftim** [comment]
/hold

@fabriziopandini you are welcome to unhold whenever you are happy there's no more feedback to account for

File: content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md
Comment (fabriziopandini) [reply content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md:4]: Fixed to the 1.4 release date

**Comment by sftim** [comment]
@pacoxu you're welcome to LGTM this.

LGTM for blog team

**Comment by pacoxu** [comment]
/lgtm 
/woof 
😄

**Pull request merged**
